recipes/_compute_base_config.rb

```diff
@@ -55,6 +55,12 @@
   end
 end
 
+# Only run FSx on centos for now
```

Comment:
why not moving this block in _compute_base_config.rb and also the block in _master_base_config.rb into base_config.rb?

Reply:
done